libraries/AC_Avoidance/AP_OAPathPlanner.cpp

```diff
@@ -57,9 +58,16 @@ const AP_Param::GroupInfo AP_OAPathPlanner::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("MARGIN_MAX", 3, AP_OAPathPlanner, _margin_max, OA_MARGIN_MAX_DEFAULT),
 
+    // @Param: RESET_ORIGIN
+    // @DisplayName: Reset way point origin to current location while recovering from Object Avoidance
+    // @Description: Enabling this would reset the origin of the waypoint to the present location while recovering from Object Avoidance. Disabling it will make the vehicle return to its original waypoint trajectory.   
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("RESET_ORIGIN", 4, AP_OAPathPlanner, _wp_reset, OA_WP_RESET_DEFAULT),
+
     // @Group: DB_
     // @Path: AP_OADatabase.cpp
-    AP_SUBGROUPINFO(_oadatabase, "DB_", 4, AP_OAPathPlanner, AP_OADatabase),
+    AP_SUBGROUPINFO(_oadatabase, "DB_", 5, AP_OAPathPlanner, AP_OADatabase),
```

Comment:
You cant change this number, it will reset the params to default, leave this as 4 and add the new one in as 5

Reply:
I have fixed it. Thanks for the catch!